src/components/readBtn.tsx

```diff
@@ -0,0 +1,12 @@
+export default function ReadBtn(): JSX.Element {
```

Comment:
I think you should add this to our generic button component `src/components/ui/button.tsx` instead of making a new file for it.

Reply:
Okay, thank you for the review!